test/bin/manage_brew_rpms.sh

```diff
@@ -51,7 +59,12 @@ action_download() {
     # cannot be identified easily when running in a CI job
     for arch in x86_64 aarch64 ; do
         local adir
-        adir="${dir}/${arch}"
+        # shellcheck disable=SC2001
+        if [ -z "${ver_type}" ] ; then
+            adir="${dir}/${arch}"
+        else
+            adir="${dir}/$(echo "${package}" | sed 's/.*microshift-\([^-]*\).*/\1/')/${arch}"
```

Comment:
Can we split this expression into another var? 
Please, explain in the comment what we're trying to extract.